Mathlib/Algebra/BigOperators/Group/List/Basic.lean

```diff
@@ -508,7 +497,8 @@ lemma drop_sum_flatten (L : List (List α)) (i : ℕ) :
   · simp
   · cases i <;> simp [take_append, *]
 
-@[deprecated (since := "2024-10-15")] alias drop_sum_join := drop_sum_flatten
+@[deprecated (since := "2024-10-25")] alias take_sum_join' := take_sum_flatten
+@[deprecated (since := "2024-10-25")] alias drop_sum_join' := drop_sum_flatten
```

Comment:
How did these get (re?)added?

Reply:
Ctrl + F for them in the diff. They come from another file